### ISSUE TITLE
Minimum access level/scope for Pinata API token?

### ISSUE THREAD

**Issue by KEINOS** [aquiladev/ipfs-action#26]
**TL; DR:** What is the minimum scope of the Pinata API token for this action?

**TS; DR:**

Thank you for providing such a nice tool!

My purpose to use this action is to push a directory of JSON contents to IPFS via Pinata if the CI passes. 

While creating a new API token in Pinata requires the scope of the API to access. **It works enabling them all**. Though, I would like to know what are the minimum scope to use this action.

I searched the docs but I couldn't find it in the[ README](https://github.com/aquiladev/ipfs-action/tree/c3ea0d17835537338f3b92e3c4c1db9607ffef0a#3-pinata-starter) nor in the [docs of IPFS](https://dapps-delivery-guide.readthedocs.io/en/latest/hosting/ipfs.html#upload-to-ipfs-pinata-pinning-service)...

Thank you in advance.

<kbd><img width="335" alt="screenshot 2021-12-01 11 58 51" src="https://user-images.githubusercontent.com/11840938/144164011-8966e5fa-26ee-4257-80f7-e5d8a8aaef0e.png"></kbd>

**Comment by aquiladev** [comment]
Thanks for the question.
Pinata is growing, previously there were not permission options.
You must select at least `pinFileToIPFS`. I checked it in the [pipeline run](https://github.com/aquiladev/dapps-delivery-guide/runs/4376908503?check_suite_focus=true)

**Comment by KEINOS** [comment]
@aquiladev Thank you for the quick response!

> You must select at least `pinFileToIPFS`. 

That's brilliant! 
I will close this issue as soon as I check, and will PR the doc as well! 👍

**Comment by KEINOS** [comment]
**I confirm that it works with `pinFileToIPFS`** only scope!

Closing the issue. Thank you!

P.S.
I will PR the updated README to this repo if I may.
